VocaDbWeb/Views/SongList/Featured.cshtml

```diff
@@ -23,7 +23,7 @@
 <div id="tabs" class="js-cloak" data-bind="show">
 	<ul>
 		@foreach (var cat in categories) {
-			<li><a href="@("#" + cat.Id + "Tab")" data-bind="click: function() { initCategory('@cat.Id'); }">@cat.Name</a></li>

```

Comment:
What was the reason for removing this? Not needed?

Reply:
The `initCategory` function doesn't exist anymore. Please see: #525